pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
By the way, hardware level interrupts don't normally use the IF or equivalent flag. They just trigger if the input is LOW at the moment of checking for interrupt states in hardware. Therefore when it comes time to trigger your interrupt routine if the input isn't in the active state then it shouldn't trigger the ISR.